src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +19,20 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final double VOLTAGE = 12; //[J]
+
+    public static class ColorWheel {
+
+        //RGB values (0-255)
+        public static final Color RED_TARGET = ColorMatch.makeColor(255, 0, 0);
+        public static final Color GREEN_TARGET = ColorMatch.makeColor(0, 255, 0);
+        public static final Color BLUE_TARGET = ColorMatch.makeColor(0, 0, 255);
+        public static final Color YELLOW_TARGET = ColorMatch.makeColor(255, 255, 0);
```

Comment:
```suggestion
        public static final Color RED_TARGET = Color.kRed;
        public static final Color GREEN_TARGET = Color.kLime;
        public static final Color BLUE_TARGET = Color.kBlue;
        public static final Color YELLOW_TARGET = Color.kYellow;
```
not required, just a suggestion